src/DB/DumpGenerator.php

```diff
@@ -106,35 +237,37 @@ public function create(bool $removeDefiners)
         }
 
         try {
-            if (flock($lockFileHandle, LOCK_EX)) {
-                $this->logger->info('Start creation DB dump...');
-
-                $command = 'timeout ' . self::DUMP_TIMEOUT . ' ' . $this->dump->getCommand();
-                if ($removeDefiners) {
-                    $command .= ' | sed -e \'s/DEFINER[ ]*=[ ]*[^*]*\*/\*/\'';
-                }
-                $command .= ' | gzip > ' . $dumpFile;
-
-                $process = $this->shell->execute('bash -c "set -o pipefail; ' . $command . '"');
-
-                if ($process->getExitCode() !== ShellInterface::CODE_SUCCESS) {
-                    $this->logger->error('Error has occurred during mysqldump');
-                    $this->shell->execute('rm ' . $dumpFile);
-                } else {
-                    $this->logger->info('Finished DB dump, it can be found here: ' . $dumpFile);
-                    fwrite(
-                        $lockFileHandle,
-                        sprintf('[%s] Dump was written in %s', date("Y-m-d H:i:s"), $dumpFile) . PHP_EOL
-                    );
-                    fflush($lockFileHandle);
-                }
-                flock($lockFileHandle, LOCK_UN);
-            } else {
+            if (!flock($lockFileHandle, LOCK_EX)) {
                 $this->logger->info('Dump process is locked!');
+                return;
             }
-            fclose($lockFileHandle);
-        } catch (\Exception $e) {
-            $this->logger->error($e->getMessage());
```

Comment:
Why did you remove error logging?

Reply:
@oshmyheliuk  error logging will occur in [Magento\MagentoCloud\Command\ DbDump::execute()](https://github.com/magento/ece-tools/pull/673/files#diff-c1669aa33365b724fea44b5e9261d06cR155)